src/offcanvas/offcanvas-backdrop.ts

```diff
@@ -32,6 +34,7 @@ import { OffcanvasDismissReasons } from './offcanvas-dismiss-reasons';
 export class NgbOffcanvasBackdrop implements OnInit {
 	private _nativeElement = inject(ElementRef).nativeElement as HTMLElement;
 	private _zone = inject(NgZone);
+	private _injector = inject(Injector);
```

Comment:
I was thinking, if we don't really need it anywhere, can we maybe just move it to the constructor?


I think these are equivalent, right?

```ts

private _injector = inject(Injector);

ngOnInit() {
  afterNextRender(() => {...}, {phase: ..., injector: this._injector})
}
```

```ts
constructor() {
  afterNextRender(() => {...}, {phase})
}
```

Reply:
Thanks for the review. I'm on holidays, so if you want to move this forward, don't hesitate to do the suggested refactorings. I won't be able to do them. Not before at least a week.

Reply:
Nice, have a good one!
I'll probably do it after or it will fix itself when moving to signals anyway.